src/crab_utils/stats.cpp

```
@@ -98,9 +95,11 @@ void CrabStats::reset() {
 }
 
 void CrabStats::count(const std::string& name) { ++(*counters)[name]; }
-void CrabStats::count_max(const std::string& name, unsigned v) { (*counters)[name] = std::max((*counters)[name], v); }
+void CrabStats::count_max(const std::string& name, const unsigned v) {
```

cppcheck

[style] The function 'count_max' is never used.

(unusedFunction)